tests/clds_hash_table_int/clds_hash_table_int.c

```diff
@@ -1317,4 +1349,241 @@ TEST_FUNCTION(clds_hash_table_set_value_with_the_same_value_succeeds_with_initia
     clds_hazard_pointers_destroy(hazard_pointers);
 }
 
+static bool get_item_and_change_state(CHAOS_TEST_ITEM_DATA* items, int item_count, LONG new_item_state, LONG old_item_state, int* selected_item_index)
+{
+    int item_index = (rand() * (item_count - 1)) / RAND_MAX;
```

Comment:
>rand [](start = 22, length = 4)

no srand?

Also, on that note, I recently found that the state of rand on Windows is thread-local. So spinning N threads that just print rand will print the same sequence in each thread, unless they each call srand with some additional entropy (seeding them each with current time might not be that different).

Maybe this is all fine for this test, so /consider.

Reply:
Added srand for each chaos test thread. Cheers!

---
In reply to: [412370500](https://github.com/dcristoloveanu/clds/pull/45#discussion_r412370500) [](ancestors = 412370500)